feature-libs/product-configurator/rulebased/components/attribute/types/checkbox/configurator-attribute-checkbox.component.ts

```diff
@@ -48,12 +48,18 @@ export class ConfiguratorAttributeCheckBoxComponent
 
   protected assembleSingleValue(): Configurator.Value[] {
     const localAssembledValues: Configurator.Value[] = [];
+    const value = this.attribute.values ? this.attribute.values[0] : undefined;
```

Comment:
why we cannot write like this: 
`const value = this.attribute?.values ? this.attribute?.values[0] : undefined;`
with the question marks

Reply:
Because attribute is mandatory, so the question marks are not needed